python/tvm/topi/x86/conv2d_int8.py

```diff
@@ -120,7 +120,7 @@ def _pack_data(cfg, data, kernel):
     kernel = te.compute(
         (oc_chunk, ic_chunk, kh, kw, ic_bn // n_elems, oc_bn, n_elems),
         lambda occ, icc, k_h, k_w, icbc, ocb, icbb: kernel[
-            occ * oc_bn + ocb, icc * ic_bn + icbc * ic_bn // n_elems + icbb, k_h, k_w
+            occ * oc_bn + ocb, icc * ic_bn + icbc * n_elems + icbb, k_h, k_w
```

Comment:
cc @tkonolige  please have a look at this change. Since `test_topi_conv2d_int8.py` doesn't use the alter layout code (which had a bug), the reason aarch64 CI failed on `test_topi_conv2d_int8.py` was probably due to this bug.